INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 134
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:88 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:88 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:88 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:88 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:88 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:88 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 137.0.7151.68 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 137.0.7151.68 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 137.0.7151.68
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 137.0.7151.70
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 10844 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:40047
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element/f.C7BC4A038A589397FDA00A2FE7D34D13.d.3487C3A5A66E98722FEA28EA79402352.e.2/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element/f.C7BC4A038A589397FDA00A2FE7D34D13.d.3487C3A5A66E98722FEA28EA79402352.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element/f.C7BC4A038A589397FDA00A2FE7D34D13.d.3487C3A5A66E98722FEA28EA79402352.e.6/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 504dafc3dce9800490924d9cb468c162a9d595b2fd60ba032fbb6e2afa9207f8
DEBUG    LoginPage:login_page.py:32 login crumb: 504dafc3dce9800490924d9cb468c162a9d595b2fd60ba032fbb6e2afa9207f8
DEBUG    MainPage:ui_element.py:69 Click on locator ('link text', 'Manage Jenkins')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/element/f.C7BC4A038A589397FDA00A2FE7D34D13.d.8CE10A9A3D2D0B0D0D8F00DD5221CC93.e.46/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element/f.C7BC4A038A589397FDA00A2FE7D34D13.d.8CE10A9A3D2D0B0D0D8F00DD5221CC93.e.46/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/url HTTP/1.1" 200 0
DEBUG    ManageJenkinsPage:ui_element.py:69 Click on locator ('xpath', "//a[@href='load-statistics']/..")
DEBUG    ManageJenkinsPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//a[@href='load-statistics']/.."),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/element/f.C7BC4A038A589397FDA00A2FE7D34D13.d.EE718629CA2F8C3CE47F43CEDF4852D4.e.81/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "POST /session/3ca1dc0ad17f90b602af5b954297ae62/element/f.C7BC4A038A589397FDA00A2FE7D34D13.d.EE718629CA2F8C3CE47F43CEDF4852D4.e.81/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "GET /session/3ca1dc0ad17f90b602af5b954297ae62/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:40047 "DELETE /session/3ca1dc0ad17f90b602af5b954297ae62 HTTP/1.1" 200 0